### ISSUE TITLE
Bagnon not working in Cata Pre-Patch

### ISSUE THREAD

**Issue by aristoblis** [Jaliborc/Bagnon#1892]
**Which software were you running?**
 - Addon version name: Bagnon
 - Client used: Bagnon-10.2.25

**Have you read the changelog?**
 Mark if read.

**Please describe the bug.**
Bagnon/Bagbrother do not work in Classic Cata Pre-Patch even tho they show as compatible with game version 4.4.0 in the Curseforge client.  The addons don't even appear in the addon list so there is no option to enable them at all.

**Please describe how to reproduce it.**
1. Install version 10.2.25 from Curseforge
2. Log into game to character select screen
3. Click on Addons
4. Bagnon/BagBrother do not appear in the list of addons, even with "Load out of date addons" enabled.

**Any Screenshots?**
![image](https://github.com/Jaliborc/Bagnon/assets/16908053/9f6d2560-d717-4e8c-9162-714b4dbbf02f)

**Error Logs are Important!**
There are no error logs because the addon never loads.

**Comment by youngr13uiuedu** [comment]
I am also getting the same thing.  I even uninstalled, and reinstalled but that did not fix the issue.

**Comment by Ifartstarz** [comment]
Yup same, not even listed in addon panel.  I hope ya can fix this soon, stock blizz bags are trash lol.

**Comment by kkerchmar** [comment]
This is an easy fix, duplicate the Wrath toc files in each of the Bagnon/Bagbrother addon folders and rename the Wrath to Cata in the filename.

**Comment by eriseil** [comment]
For BagBrother and each Bagnon addon, duplicate the `<addon name>-Wrath.toc` as `<addon name>-Cata.toc`. Update the interface version in each of those to 40400. Not sure what other changes might be relevant to a big update like this, but if you just want it to work until the next release, this is what I did and mine is working now.

**Comment by ffvixrefugee** [comment]
> For BagBrother and each Bagnon addon, duplicate the `<addon name>-Wrath.toc` as `<addon name>-Cata.toc`. Update the interface version in each of those to 40400. Not sure what other changes might be relevant to a big update like this, but if you just want it to work until the next release, this is what I did and mine is working now.

worked for me thanks boss

**Comment by oldspices** [comment]
> For BagBrother and each Bagnon addon, duplicate the `<addon name>-Wrath.toc` as `<addon name>-Cata.toc`. Update the interface version in each of those to 40400. Not sure what other changes might be relevant to a big update like this, but if you just want it to work until the next release, this is what I did and mine is working now.

Where exactly is this at? Im not seeeing anything including the word "wrath"

**Comment by nyan-left** [comment]
@oldspices 

You can find it in `/Interface/AddOns/Bagnon`

**Comment by shellt1** [comment]
Worked like a charm thanks for the fix

**Comment by aristoblis** [comment]
After the 10.2.26 update mine is working fine now.  Thanks!
